atd-etl/cris_import/cris_import.py

```diff
@@ -66,10 +67,22 @@ def main(cli_args):
         )
 
         if cli_args.pdf and not no_crashes_found:
-            pdfs_processed_count = process_pdfs(
-                extract_dir, cli_args.s3_upload, cli_args.workers
-            )
-            records_processed["pdfs"] = pdfs_processed_count
+            """
+            Make sure the crashReports directory exists. It may be missing if there were no
+            crashes to process. It may also be missing if the extract was not configured
+            to include CR3 PDFs
+            """
+            crash_reports_dir = os.path.join(extract_dir, "crashReports")
+
+            if os.path.exists(crash_reports_dir):
+                records_processed["pdfs"] = process_pdfs(
+                    extract_dir, cli_args.s3_upload, cli_args.workers
+                )
+            else:
+                logger.warning(
+                    f"Warning: No 'crashReports' directory found — no PDFs to process"
```

Comment:
Yep, i saw this warning as well as the one about mismatch between crash count and pdf count with the test extracts.
```
WARNING: Warning: No 'crashReports' directory found — no PDFs to process
WARNING: Warning: there was a mismatch between the # of CSV crashes processed (2493) vs the CR3 PDFs processed (0).
```